### PR TITLE
nfdump: add nfcapd service script

### PR DESCRIPTION
**Pull request by ailiop-git**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [ ] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/nfdump/files/nfcapd/run [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+[ -r conf ] && . ./conf
+exec nfcapd -u _nfcapd -g _nfcapd -l /var/log/nfdump $OPTS
```
Comment (abenson) [inline srcpkgs/nfdump/files/nfcapd/run:3]: Could `/var/log/nfdump` be changed to ${LOGDIR:=/var/log/nfdump} to make it easier to override the destination?

Comment (Duncaen) [reply srcpkgs/nfdump/files/nfcapd/run:3]: Should probably just be `${OPTS:--l /var/log/nfdump}`.

**Comment by abenson** [comment]
The tests seem to fail on time-based issues, not really sure what's going on.

**Comment by ailiop-git** [comment]
> The tests seem to fail on time-based issues, not really sure what's going on.

Probably unmaintained tests, haven't really looked into it since it is otherwise working fine. The test failure has always been there anyway, it's just that this is the first update after enabling the checking stages.

**Comment by ericonr** [comment]
@Duncaen or @abenson, ok to merge?

**Comment by abenson** [comment]
I feel comfortable yeah

**Pull request closed without merging**